assets/css/main.css

```diff
@@ -653,13 +653,14 @@ ul li {
 }
 
 .startups img {
-  -webkit-filter: contrast(60%); /* Safari */
-  filter: contrast(60%);
+  transition: all .2s linear;
```

Comment:
You're editing CSS files again. So, change the version number in the html file 🙃